tests/lib/rules/no-control-regex.js

```diff
@@ -28,6 +28,7 @@ ruleTester.run("no-control-regex", rule, {
         "new (function foo(){})('\\x1f')"
     ],
     invalid: [
+        { code: "var regex = /\\x1f/", errors: [{ message: "Unexpected control character in regular expression.", type: "Literal"}] },
         { code: "var regex = /\\\x1f/", errors: [{ message: "Unexpected control character in regular expression.", type: "Literal"}] },
```

Comment:
Oh you are right, double backslash will make it escaped. I will move it to the valid list. Need to rework on the checks too. Thanks for feedback.